docs/src/main/asciidoc/security-jpa-concept.adoc

```diff
@@ -0,0 +1,180 @@
+[id="security-jpa-concept"]
+= Quarkus Security with JPA
+include::_attributes.adoc[]
+:categories: security,jpa
+
+JPA is an identity provider, similar to xref:security-jdbc.adoc[JDBC], suitable for use with the xref:security-basic-authentication-concept.adoc[Basic] and xref:security-authentication-mechanisms-concept.adoc#form-auth[Form-based] Quarkus security mechanisms, which require a combination of username and password credentials.
```

Comment:
I think the _**s**_ in security should be uppercase here and elsewhere when referring to the noun "Quarkus Security", which is the name of the Quarkus framework for security.

Reply:
Great catch @michelle-purcell . Fixing here and on some other places too.